### PR TITLE
Check all must-gather commads by versions

### PR DESCRIPTION
**Pull request by AviadP**
- Added function that that return ocs version number as major.minor (float)
- Added new constant with list of must gather commands, base and extended
- Checks must-gather commands according to OCS current version
- Added bugzilla marker

### CODE REVIEW

File: ocs_ci/ocs/ocp.py [hunk 1]
```diff
@@ -926,6 +926,22 @@ def get_ocs_version():
     return ocp_cluster.get()['items'][0]['spec']['version']
 
 
+def get_ocs_parsed_version():
+    """
+    Get ocs version as float
+
+    Returns:
+        float: ocs version number as major.minor (for example: 4.5)
```
[the remark this answers is not in the document]
Comment (ebenahar) [reply ocs_ci/ocs/ocp.py:934]: oh sorry, now I see it is `Returns`

**Review by ebenahar — APPROVED** [review]

**Comment by AviadP** [comment]
validation failed due to this bug: https://bugzilla.redhat.com/show_bug.cgi?id=1885640

File: .gitignore [hunk 2]
```diff
@@ -19,3 +19,8 @@ data/*
 external/
 apidoc/*
 _build/*
```
Comment (ebenahar) [inline .gitignore:21]: are these changes intentional?

Comment (AviadP) [reply .gitignore:21]: no

**Comment by AviadP** [comment]
closing this PR, @OdedViner will implement another solution for this issue, that also cover 4.6
